src/ucp/api/ucp_def.h

```diff
@@ -731,6 +731,15 @@ typedef struct ucp_ep_params {
      */
     ucs_sock_addr_t         sockaddr;
 
+    /**
+     * Local address in the form of a sockaddr; 
+     * Forces using a specific network device for the new connection.
+     * This value is optional. This field should be set
+     * along with its corresponding bit in the field_mask - @ref
+     * UCP_EP_PARAM_FIELD_LOCAL_SOCK_ADDR.
+     */
```

Comment:
I found this description didn't scan well.  Suggest here and in other places with something along the lines of

Local address in the form of a sockaddr; specifies the associated network device for the new connection.   This setting is optional.  To enable it, the corresponding @ref * UCP_EP_PARAM_FIELD_LOCAL_SOCK_ADDR bit in the field_mask must be set.

Reply:
fixed accordingly